packages/renderer/MdToHtml/rules/mermaid.ts

```diff
@@ -11,6 +11,12 @@ export default {
 			{
 				name: 'mermaid_render.js',
 			},
+			{
+				inline: true,
+				// This properly renders the fa-icons in both the Rich Text Editor and Markdown Pane / Read-Only Mode
+				text: '@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css");',
```

Comment:
Please avoid importing files from a CDN — in general, Joplin should avoid network connections that aren't authorized by a user (and be functional offline). 

An alternative option could be to make `font-awesome`'s `css` files a plugin asset (similar to the CSS included by the KaTeX renderer plugin).

Thank you for working on this!